src/test/kotlin/io/github/enyason/predictions/PredictionsApiTest.kt

```diff
@@ -0,0 +1,90 @@
+package io.github.enyason.predictions
+
+import com.google.gson.Gson
+import io.github.enyason.base.ReplicateConfig
+import io.github.enyason.client.TestPredictable
+import io.github.enyason.domain.mappers.toPrediction
+import io.github.enyason.predictions.models.PredictionDTO
+import kotlinx.coroutines.test.runTest
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+
+class PredictionsApiTest {
+
+    private lateinit var mockWebServer: MockWebServer
+    private lateinit var sut: PredictionsApi
+    private val gson = Gson()
+
+    @BeforeTest
+    fun setup() {
+        mockWebServer = MockWebServer()
+        mockWebServer.start(8080)
+        sut = PredictionsApi(ReplicateConfig("tokenYUIOPJHIGUFYDRT", baseUrl = mockWebServer.url("").toString()))
```

Comment:
why is the baseUrl set from the mockWebServer.url with an empty string?

Reply:
The url functions accepts a path which should either be an empty string or **/**